com.unity.cinemachine/Runtime/Behaviours/CinemachineGroupFraming.cs

```diff
@@ -160,7 +160,7 @@ public void Reset()
                 return;
 
             var group = vcam.FollowTargetAsGroup;
-            if (group == null || group.Sphere.radius < k_MinimumGroupSize)
+            if (group == null)
```

Comment:
I don't like this fix, because it has a side-effect: it will now attempt to frame a zero-size group, which it should not do.
Maybe a better fix would be to have our own PreviousStateIsValid flag in ExtraData.  Clear it in extra.Reset().